Tests/SHLLMTests/Models/Qwen2_5-7BTests.swift

```diff
@@ -19,3 +19,47 @@ func canLoadAndQueryQwen2_5__7B() async throws {
     Swift.print(result)
     #expect(!result.isEmpty)
 }
+
+@Test
+func canHelpMeFetchTheWeatherWithQwen2_5__7B() async throws {
+    guard let llm = try await Qwen2_5__7B.tests else { return }
+
+    let tool1: WeatherTool = try await llm.request(
+        tools: Tools([weatherToolFunction]),
+        messages: [
+            [
+                "role": "system",
+                "content": "You are a weather fetching assistant. Your only purpose is to fetch weather data.",
+            ],
+            ["role": "user", "content": "What is weather in Paris like?"],
+        ]
+    )
+
+    let expectedTool1 = WeatherTool.getCurrentWeather(.init(
+        location: "Paris, France",
+        unit: .celsius
```

Comment:
@myobie Interestingly, Qwen 2.5 **7B** usually seems to return Celsius for this question. Granted, it makes sense because France uses Celsius. The smaller version of this model usually returns Fahrenheit, though, maybe because it's not smart enough to think about what measurement people in the city being asked about uses. Of course, that's just a guess.